src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -493,6 +493,16 @@ public static class UIElement
 #endif
 		}
 
+		public static class VisualState
+		{
+			/// <summary>
+			/// When this is set, the <see cref="Windows.UI.Xaml.VisualState.Setters"/> will be applied synchronously when changing state,
+			/// unlike UWP which waits the for the end of the <see cref="VisualTransition.Storyboard"/> (if any) to apply them.
+			/// </summary>
+			/// <remarks>This flag is for backward compatibility with old versions of uno and should not be turned on.</remarks>
```

Comment:
Yeah : I expect that it would break some applications that are relying on an invalid behavior. And it's really easy to support it, no needs to break apps for fun ;)